ts/Core/Axis/Color/ColorAxis.ts

```diff
@@ -572,13 +573,29 @@ class ColorAxis extends Axis implements AxisLike {
         ).attr({
             zIndex: 1
         }).add(item.legendGroup);
-
+        const options = legend.options;
+        const symbolPadding = options.symbolPadding || 0;
+        const isSeries = !(item as any).series;
+        const series = !isSeries && (item as any).series.drawLegendSymbol ?
+                (item as any).series :
+                item,
+            seriesOptions = series.options,
+            showCheckbox = legend.createCheckboxForItem &&
+                seriesOptions &&
+                seriesOptions.showCheckbox;
+        const itemExtraWidth = legend.symbolWidth + symbolPadding +
+                        itemDistance + (showCheckbox ? 20 : 0);
```

Comment:
A consistent format would look better - not a mix of `const` in each line and a comma-separated list. Pick one and fight [here](https://highsoft.slack.com/archives/C2SD4K2MN/p1631862207052200).